python/lsst/cp/pipe/ptc/cpSolvePtcTask.py

```diff
@@ -174,11 +174,21 @@ class PhotonTransferCurveSolveConfig(pipeBase.PipelineTaskConfig,
             "otherwise ADU.",
         default=50_000.,
     )
+    maxDeltaInitialPtcOutlierFit = pexConfig.Field(
+        dtype=float,
+        doc="If there are any outliers in the initial fit above "
+            "maxSignalInitialPtcOutlierFit, then no points that have this delta "
+            "from the previous ``good`` point are allowed. If "
```

Comment:
Can you clarify that this delta is in terms of the `rawMeans` vector (or `finalMeans`).  The units being to first order implies this (so that it's not based on the variance).